perf_dashboard/bq_utils.py

```diff
@@ -22,25 +22,22 @@
 def insert_rows(project, dataset_name, table_name, rows):
     """Insert rows to bigquery table."""
     client = bigquery.Client(project=project)
-    dataset = client.dataset(dataset_name)
-    table = bigquery.Table(table_name, dataset)
-    table.reload()
-    table.insert_data(rows)
-
+    dataset_ref = client.dataset(dataset_name)
+    table_ref = dataset_ref.table(table_name)
+    table = client.get_table(table_ref)
+    client.create_rows(table, rows)
 
 def execute_query(query):
     """Execute query and return the query results."""
     client = bigquery.Client()
-    query_job = client.run_async_query(str(uuid.uuid4()), query)
-    query_job.use_legacy_sql = False
+    query_job = client.query((query))
 
     # Start the query job and wait it to complete
-    query_job.begin()
-    query_job.result()
+    data = query_job.result()
+
+    results = []
 
-    # Get the results
-    destination_table = query_job.destination
-    destination_table.reload()
-    results = destination_table.fetch_data()
+    for row in data:
```

Comment:
why not just return `list(query_job.result())`?

Reply:
This will return a list of `Row` object, but I need tuples to add timestamp to each tuple and `client.create_rows` only accept list of tuples or list of dicts.

Reply:
return [row.values for row in query_job.result()]?

Reply:
Done, thanks for the suggestion :+1: